playbooks/roles/topbeat/templates/topbeat.yml.j2

```diff
@@ -0,0 +1,19 @@
+---
+input:
+  # In seconds, defines how often to read server statistics
+  period: {{ topbeat_period_seconds }}
+
+  # Regular expression to match the processes that are monitored
+  # By default, all the processes are monitored
+  procs: [".*"]
+
+  # Statistics to collect (all enabled by default)
```

Comment:
This is a catch all for everything topbeat can track. If it can track it it will. This is the default.